<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="user-PC" failures="3" tests="4" name="test" time="49.1" errors="0" timestamp="2023-07-17T19:51:49 IST">
  <testcase classname="testCases.LoginDataDrivenTest" name="testLogin" time="6.383">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.LoginDataDrivenTest.testLogin(LoginDataDrivenTest.java:62)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
  <testcase classname="testCases.LoginDataDrivenTest" name="testLogin" time="4.341">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.LoginDataDrivenTest.testLogin(LoginDataDrivenTest.java:62)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
  <testcase classname="testCases.LoginDataDrivenTest" name="testLogin" time="12.408"/>
  <testcase classname="testCases.LoginDataDrivenTest" name="testLogin" time="1.768">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.LoginDataDrivenTest.testLogin(LoginDataDrivenTest.java:72)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
</testsuite> <!-- test -->
